dts/bindings/gpio/st,stm32-gpio.yaml

```diff
@@ -20,6 +20,24 @@ properties:
     "#gpio-cells":
       const: 2
 
+    remap-pa11:
+      type: boolean
+      required: false
+      description: Remaps the PA11 pin to operate as PA9 pin.
+        Use of this property is restricted to GPIO port A on STM32G0 SoCs.
+
+    remap-pa12:
+      type: boolean
+      required: false
+      description: Remaps the PA12 pin to operate as PA10 pin.
+        Use of this property is restricted to GPIO port A on STM32G0 SoCs.
+
+    remap-pa11-pa12:
+      type: boolean
+      required: false
+      description: Remaps the PA11/PA12 pin to operate as PA9/PA10 pin.
+        Use of this property is restricted to GPIO port A on STM32F070x SoCs.
```

Comment:
I wonder if these properties should be part of `pinctrl` compatible instead of `gpio`, since it's the pinmux/pinctrl driver who's responsible for handling them.

Reply:
Good point, that would be cleaner than GPIO